### ISSUE TITLE
Should have a way to create pages for types

### ISSUE THREAD

**Issue by wch** [machow/quartodoc#51]
For the Shiny docs, we want pages (or perhaps one combined page) of types. For example, in the `input_slider` page, there are many types which are not meaningful to users, and they don't have a good way to figure out what they are.

<img width="842" alt="image" src="https://user-images.githubusercontent.com/86978/222843371-fced653a-5d12-43ac-bcd7-05478ee65036.png">

**Comment by machow** [comment]
Attributes can be documented using this syntax.

For example, if you have the file `example_file.py`:

```python
some_type = Union[bool, int]
"""This type represents the union of a bool or int"""
```

It finds the docstring:

```python
from quartodoc import get_object

print(get_object("example_file", "some_type").docstring.value)
```

**Comment by machow** [comment]
Everything in `shiny.types` can be on its own page (i.e. 1 page for all types).

**Comment by machow** [comment]
Plan:

* Get relevant `shiny.types` onto a page
* Flag other types that we want to link out to
* Flag type names we want to change (e.g. should all types from the python `typing` module not have the word `typing.`? e.g. `bool` rather than `typing.bool`; what about htmltools, etc..?)
* Modify shiny renderer to render annotations of interest to markdown links.

**Comment by machow** [comment]
Fixed by #64
